components/Stats.vue

```diff
@@ -245,6 +309,19 @@ export default {
     },
     contains(arr, item) {
       return arr.includes(item)
+    },
+    onChange(event) {
+      const countyName = event.target.value
+      const countyNameIndex = getCountyShortName(countyName)
+
+      // HACK TO WORKAROUND AGE/GENDER DATA MISMATCH BUG
+      if (this.DataVTwo[countyNameIndex]) {
+        this.CountyDataVTwo = this.getFormatData()
+      }
+    },
+    getFormatData() {
+      const allCounties = Object.keys(Data)
```

Comment:
Is there any reason to get county list from Data which is Data v1?
We want to take over v1 to use v2 and remove v1 in near future once v2 is completely ready, so we better not depend on v1 and v2 each other. Getting county list Data v2 sounds good here.

Reply:
v2 doesn't have the full list of counties. It just has the counties with available data.

Reply:
ahh, I see.